redis/service.cc

```diff
@@ -52,12 +52,14 @@ future<> redis_service::listen(seastar::sharded<auth::service>& auth_service, db
     auto family = cfg.enable_ipv6_dns_lookup() || preferred ? std::nullopt : std::make_optional(net::inet_address::family::INET);
     auto ceo = cfg.client_encryption_options();
     auto keepalive = cfg.rpc_keepalive();
-    redis_transport::redis_server_config redis_cfg;
-    redis_cfg._timeout_config = make_timeout_config(cfg);
-    redis_cfg._read_consistency_level = make_consistency_level(cfg.redis_read_consistency_level());
-    redis_cfg._write_consistency_level = make_consistency_level(cfg.redis_write_consistency_level());
-    redis_cfg._max_request_size = memory::stats().total_memory() / 10;
-    redis_cfg._total_redis_db_count = cfg.redis_database_count();
+    redis_transport::redis_server_config redis_cfg {
+        make_timeout_config(cfg),
+        memory::stats().total_memory() / 10,
+        make_consistency_level(cfg.redis_read_consistency_level()),
+        make_consistency_level(cfg.redis_write_consistency_level()),
+        cfg.redis_database_count(),
+        ssg
+    };
```

Comment:
Why did you change this code (other then setting ssg)?
The original code was easier to understand.